### ISSUE TITLE
[Bug Report] Arithmetic Ops Fail Accuracy Checks On Large Floats

### ISSUE THREAD

**Issue by ctodTT** [tenstorrent/tt-metal#17090]
**Describe the bug**
When performing arithmetic on `ttnn` tensors, accuracy breaks down, reminiscent of precision failures. This occurs even when using an identity operation, such as multiplying by 1

**To Reproduce**
1. Build `tt-metal`
2. Create a file `precision_test.py`:
```python
import torch
import ttnn

device_id = 0
device = ttnn.open_device(device_id=device_id)

a = torch.randn((5,)) * 100000000
b = torch.Tensor([1])
a_ttnn = ttnn.from_torch(a, layout=ttnn.TILE_LAYOUT, device=device)
b_ttnn = ttnn.from_torch(b, layout=ttnn.TILE_LAYOUT, device=device)

print(f"shape: {a_ttnn.shape}")
print(f"layout: {a_ttnn.layout}")
print(f"dtype: {a_ttnn.dtype}")


print(a * b)
print(a_ttnn * b_ttnn)
```
3. Execute `python precision_test.py`
4. Note the difference in the `torch` multiplication output and the `ttnn` multiplication output. 

**Expected behavior**
The two output tensors should be close

**Additional context**
This issue is arising on the `tt-mlir` side when attempting to execute a llama attention layer through `mlir`

** Relevant People **
@tapspatel

**Comment by pgkeller** [comment]
@ctodTT is this issue still relevant?

**Comment by minaliuTT** [comment]
@tapspatel Is this issue still relevant, or can it be closed? Thanks!

**Comment by tapspatel** [comment]
we can close, we are not tracking it internally. thanks
